cpr/accept_encoding.cpp

```diff
@@ -0,0 +1,23 @@
+#include "cpr/accept_encoding.h"
+
+namespace cpr {
+
+AcceptEncoding::AcceptEncoding(const std::initializer_list<AcceptEncodingMethods>& methods) : methods_{methods} {}
+
+bool AcceptEncoding::isEmpty() const {
```

Comment:
All public methods have to start with an upper case letter.

Reply:
I would like to change its name to `has`, just like the implementation in proxies.cpp:
```c++
bool Proxies::has(const std::string& protocol) const {
    return hosts_.count(protocol) > 0;
}
```

Reply:
This is also possible.